src/plugins/home/kibana.jsonc

```diff
@@ -6,8 +6,7 @@
     "id": "home",
     "server": true,
     "browser": true,
-    "requiredPlugins": ["dataViews", "share", "urlForwarding"],
-    "optionalPlugins": ["usageCollection", "customIntegrations", "cloud", "guidedOnboarding"],
-    "requiredBundles": ["kibanaReact"]
+    "requiredPlugins": ["dataViews", "share", "urlForwarding", "kibanaReact" ],
```

Comment:
Can you help me understand why `kibanaReact` was moved here?

Reply:
I was having issues running it locally but seems to be ok now - reverting back thanks!